doc/user/layouts/shortcodes/mysql-direct/mysql-schema-changes.html

```diff
@@ -0,0 +1,22 @@
+Materialize supports schema changes in the upstream MySQL database as
+follows:
+
+#### Compatible schema changes
+
+- Adding columns to tables. Materialize will **not ingest** these columns unless
```

Comment:
@rjobanp, @morsapaes: We might want to limit this. It is possible to add columns at the end but adding columns specifying the position with `FIRST` / `AFTER` won't work.